modules/comparison-of-solutions-to-provision-node-local-storage.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent_storage_local/overview-and-comparison-of-local-storage-options
+
+:_mod-docs-content-type: CONCEPT
+[id="comparison-of-solutions-to-provision-node-local-storage_{context}"]
+= Comparison of LVM Storage, LSO, and HPP
+
+[id="comparing-storage-types"]
+== Comparison of the support for storage types and filesystems
+
+The following table compares the support for storage types and filesystems provided by {lvms}, Local Storage Operator (LSO), and HostPath Provisioner (HPP) to provision local storage:
+
+.Comparison of the support for storage types and filesystems
+[cols="6a,5a,5a,5a",options="header"]
+|====
+
+|Functionality|LVM Storage|LSO |HPP
+
+|Support for block storage |Yes |Yes |No
+
+|Support for file storage |Yes |Yes |Yes
+
+|Support for object storage ^[1]^ |No |No |No
+
+|Available filesystems |`ext4`, `xfs` |`ext4`, `xfs` |Any mounted system available on the node is supported.
+
+|====
+[.small]
+--
+1. None of the solutions ({lvms}, LSO, and HPP) provide support for object storage. Therefore, if you want to use object storage, you need an S3 object storage solution, such as `MultiClusterGateway` from the Red Hat OpenShift Data Foundation. All of the solutions can serve as underlying storage providers for the S3 object storage solutions.
+--
+
+[id="comparing-core-functionalities"]
+== Comparison of the support for core functionalities
+
+The following table compares how {lvms}, Local Storage Operator (LSO), and HostPath Provisioner (HPP) support core functionalities for provisioning local storage:
+
+.Comparison of the support for core functionalities
+[cols="6a,5a,5a,5a",options="header"]
+|====
+
+|Functionality|LVM Storage|LSO |HPP
+
+|Support for automatic file system formatting |Yes |Yes |N/A
+
+|Support for dynamic provisioning |Yes |No |No
+
+|Support for using software Redundant Array of Independent Disks (RAID) arrays 
+|Yes 
+
+Supported on 4.15 and higher versions.
+
+|Yes |Yes
+
+|Support for transparent disk encryption |Yes 
+
+Supported with 4.16 and higher versions.
```

Comment:
Same comment as above, also suggestion to change to say "supported on" for consistency with the other entry but that part is up to you:
```suggestion
Supported on 4.16 or later.
```

Reply:
Updated as per the suggestion.